ios/RNInstabug/InstabugReactBridge.m

```diff
@@ -298,7 +298,14 @@ - (dispatch_queue_t)methodQueue {
                       startTime:(double)startTime
                        duration:(double)duration
                    gqlQueryName:(NSString * _Nullable)gqlQueryName
-             serverErrorMessage:(NSString * _Nullable)serverErrorMessage) {
+             serverErrorMessage:(NSString * _Nullable)serverErrorMessage
+                  w3cExternalTraceAttributes:(NSDictionary * _Nullable)w3cExternalTraceAttributes){
+   NSNumber *isW3cCaught = (w3cExternalTraceAttributes[@"isW3cHeaderFound"] != [NSNull null]) ? w3cExternalTraceAttributes[@"isW3cHeaderFound"] : nil;
+        NSNumber * partialID = (w3cExternalTraceAttributes[@"partialId"] != [NSNull null]) ? w3cExternalTraceAttributes[@"partialId"] : nil;
+        NSNumber * timestamp = (w3cExternalTraceAttributes[@"networkStartTimeInSeconds"] != [NSNull null]) ? w3cExternalTraceAttributes[@"networkStartTimeInSeconds"] : nil;
+        NSString * generatedW3CTraceparent = (w3cExternalTraceAttributes[@"w3cGeneratedHeader"] != [NSNull null]) ? w3cExternalTraceAttributes[@"w3cGeneratedHeader"] : nil;
+        NSString * caughtW3CTraceparent = (w3cExternalTraceAttributes[@"w3cCaughtHeader"] != [NSNull null]) ? w3cExternalTraceAttributes[@"w3cCaughtHeader"] : nil;
```

Comment:
there is a type mismatch between NSNumber or NSString and value of type [NSNull null] so thats why we need to explicitly check the value before we assign it to avoid unexpected behavior or potential crashes